packages/applications-service-api/src/repositories/document.backoffice.repository.js

```diff
@@ -79,8 +79,19 @@ const getDocumentsByType = async (queryData) =>
 		take: 1
 	});
 
+const getDocumentsByIds = async (documentIds) => {
+	return prismaClient.document.findMany({
+		where: {
+			id: {
+				in: documentIds.split(',').map((id) => parseInt(id))
```

Comment:
perhaps a guard against undefined `documentIds` as it's passed in without checking in https://github.com/Planning-Inspectorate/applications-service/pull/921/files#diff-c26781024639e6db023604f87cca30b9c794955792f329c0112678ec304dc249R60
```suggestion
				in: documentIds?.split(',').map((id) => parseInt(id))
```